src/platform-integration/linux/install-linux/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Add Linux devtools for Flutter
```

Comment:
What is Linux devtools? That is not a thing. There are Chrome DevTools and Dart/Flutter DevTools. I don't even understand what this means.